src/main/java/ReviewResponse.java

```diff
@@ -13,6 +16,8 @@ public class ReviewResponse {
     @Getter
     String okbb;
 
+    @Setter
+    String blabla;
```

Comment:
Directly exposing class fields like 'blabla' without methods to control access violates the Encapsulation principle of SOLID. (Severity: high)